tests/tck/features/mutate/InsertWithTimeType.feature

```diff
@@ -107,6 +107,76 @@ Feature: Insert with time-dependent types
     Then the result should be, in any order:
       | edge_date.f_date | edge_date.f_time  | edge_date.f_datetime         |
       | '2017-03-04'     | '23:01:00.000000' | '2017-03-04T22:30:40.000000' |
+    # insert with timezone offset and microseconds
+    When try to execute query:
+      """
```

Comment:
Nice pr! For full consideration of `cypher` compatibility, can you move some TCK tests from [https://github.com/opencypher/openCypher/tree/master/tck/features/expressions/temporal](https://github.com/opencypher/openCypher/tree/master/tck/features/expressions/temporal)?

Reply:
Good catch! I will add it later.

Reply:
You might open an issue for this so you won't forget.